transport/tchannel/outbound_test.go

```diff
@@ -232,12 +246,20 @@ func TestCallFailures(t *testing.T) {
 			if call.MethodString() == "unexpected" {
 				err = tchannel.NewSystemError(
 					tchannel.ErrCodeUnexpected, "great sadness")
-			} else {
+				call.Response().SendSystemError(err)
+			} else if call.MethodString() == "not a procedure" {
```

Comment:
let's use a `const unknownMethod = "not a procedure"` at the top of this function since we use this string in multiple places

Reply:
Added.

Reply:
still here :)